Packs/CommonScripts/Scripts/IncidentFields/TestData/integration_incidentfields.json

```diff
@@ -1 +1 @@
me": "Geo Country", "caseInsensitive": true, "name": "Geo Country", "ownerOnly": false, "required": false, "modified": "2019-12-30T16:43:18.120015579-05:00", "columns": null, "selectValues": null, "validationRegex": "", "sla": 0}, {"sortValues": null, "closeForm": false, "cliName": "geolocation", "neverSetAsRequired": false, "threshold": 72, "defaultRows": null, "id": "indicator_geolocation", "validatedError": "", "group": 2, "script": "", "isReadOnly": false, "commitMessage": "", "system": false, "content": true, "unsearchable": false, "version": 4, "unmapped": false, "hidden": false, "type": "shortText", "shouldCommit": false, "editForm": true, "fieldCalcScript": "", "description": "", "systemAssociatedTypes": null, "breachScript": "", "vcShouldIgnore": false, "associatedTypes": ["IP"], "locked": false, "placeholder": "", "useAsKpi": false, "associatedToAll": false, "prevName": "Geo Location", "caseInsensitive": true, "name": "Geo Location", "ownerOnly": false, "required": false, "modified": "2019-12-30T16:43:18.105798005-05:00", "columns": null, "selectValues": null, "validationRegex": "", "sla": 0}, {"sortValues": null, "closeForm": false, "cliName": "groups", "neverSetAsRequired": false, "threshold": 72, "defaultRows": null, "id": "indicator_groups", "validatedError": "", "group": 2, "script": "", "isReadOnly": false, "commitMessage": "", "system": false, "content": true, "unsearchable": false, "version": 4, "unmapped": false, "hidden": false, "type": "longText", "shouldCommit": false, "editForm": true, "fieldCalcScript": "", "description": "", "systemAssociatedTypes": null, "breachScript": "", "vcShouldIgnore": false, "associatedTypes": ["Account"], "locked": false, "placeholder": "", "useAsKpi": false, "associatedToAll": false, "prevName": "Groups", "caseInsensitive": true, "name": "Groups", "ownerOnly": false, "required": false, "modified": "2019-12-30T16:43:18.35567477-05:00", "columns": null, "selectValues": null, "validationRegex": "", "sla": 0}, {"sortValues": null, "closeForm": false, "cliName": "hostname", "neverSetAsRequired": false, "threshold": 72, "defaultRows": null, "id": "indicator_hostname", "validatedError": "", "group": 2, "script": "", "isReadOnly": false, "commitMessage": "", "system": false, "content": true, "unsearchable": false, "version": 4, "unmapped": false, "hidden": false, "type": "shortText", "shouldCommit": false, "editForm": true, "fieldCalcScript": "", "description": "", "systemAssociatedTypes": null, "breachScript": "", "vcShouldIgnore": false, "associatedTypes": ["IP", "hostname"], "locked": false, "placeholder": "", "useAsKpi": false, "associatedToAll": false, "prevName": "Hostname", "caseInsensitive": true, "name": "Hostname", "ownerOnly": false, "required": false, "modified": "2019-12-30T16:43:18.15837464-05:00", "columns": null, "selectValues": null, "validationRegex": "", "sla": 0}, {"sortValues": null, "closeForm": false, "cliName": "id", "neverSetAsRequired": false, "threshold": 72, "defaultRows": nu
```

Comment:
what is the change in this file?

Reply:
for all of these I went and found a pack (chose detectionsla randomly) and switched it to the case that caused the script to fail

Reply:
so I switched assosciatedTypes to none and assosciateAll to false